recorder/src/main/cpp/stacktraces.cc

```diff
@@ -0,0 +1,23 @@
+#include "stacktraces.hh"
+
+std::uint32_t Stacktraces::fill_backtrace(NativeFrame* buff, std::uint32_t capacity) {//TODO: write me 3 tests { (capacity > stack), (stack > capacity) and (stack == capacity) }
+    std::uint64_t rbp, rpc;
+    asm("movq %%rbp, %%rax;"
+        "movq %%rax, %0;"
+        "lea (%%rip), %%rax;"
```

Comment:
I wanted to confirm whether using %rbx in lea make any difference?
But i got the answer from "It'll have to overflow one variable to memory in that case, but it'll be served out of L1 anyway"